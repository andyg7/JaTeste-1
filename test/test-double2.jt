func int main()
{
	double d1;
	double d2;
	double d3;

	d1 = -10.1;
	d2 = 7.89;
	d3 = d1 + d2;

	if (d1 == -10.1 ) {
		print("passed");
	}

	if (d3 == -2.21) {
		print("passed");
	} else {
		print("failed");
	}

	d1 = -9.14;
	d2 = -9.14;

	if (d1 == d2) {
		print(d1);
	}

	return 0;
}
