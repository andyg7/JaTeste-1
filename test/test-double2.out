passed
passed
-9.140000
